docs/source/command_line.rst

```diff
@@ -353,12 +353,17 @@ definitions or calls.
 .. option:: --disallow-untyped-defs
 
     This flag reports an error whenever it encounters a function definition
-    without type annotations.
+    with without type annotations or with incomplete type annotations.
```

Comment:
It looks like this "with" doesn't belong.

Reply:
7c8d30597e2dd9f6ee3c0638048024bd53950564